types/tampermonkey/index.d.ts

```diff
@@ -239,6 +224,123 @@ declare namespace Tampermonkey {
     }
 
     type NotificationDetails = TextNotification | HighlightNotification;
+
+    interface GM_info {
+        downloadMode: 'native' | 'browser' | 'disabled';
+        isFirstPartyIsolation: boolean | undefined;
```

Comment:
isFirstPartyIsolation?: boolean;

Reply:
It is always present (boolean or undefined). It is not an optional property (from what I could see), and enabling --exactOptionalPropertyTypes would result in different behaviour than is the case.

Reply:
Didn't know about `exactOptionalPropertyTypes`. Okay then